Specs/Scene/ImplicitSubtreeSpec.js

```diff
@@ -30,7 +29,9 @@ describe("Scene/ImplicitSubtree", function () {
   function expectTileAvailability(subtree, availability) {
     var expectedAvailability = availabilityToBooleanArray(availability);
     for (var i = 0; i < availability.lengthBits; i++) {
-      expect(subtree.tileIsAvailable(i)).toEqual(expectedAvailability[i]);
+      expect(subtree.tileIsAvailableAtIndex(i)).toEqual(
```

Comment:
This file is missing coverage for `tileIsAvailableAtCoordinates` and `childSubtreeIsAvailableAtCoordinates`. This could be a good place to put them, but it was annoying to derive the coordinate from `i`. I wonder if there's a better way to do coverage for those new functions.

Reply:
would it help to add some static functions to `ImplicitTilingTester` if you find yourself making the same coordinate calculation over and over?

Reply:
or just a helper function in this file if this is the only place that uses it.